projects/plugins/jetpack-boost/bin/generate-release-zip.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Shouldn't be needed in the monorepo.

Reply:
https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025